ondewo/logging/decorators.py

```diff
@@ -100,6 +100,9 @@ def stop(self, func_name: Optional[str] = None) -> float:
         assert self._start_time
         elapsed_time = time.perf_counter() - self._start_time
 
+        # Reset the start time
+        self._start_time = None
```

Comment:
I dared to add this to Timer, so the elapsed time is calculated correctly in repeated calls of the decorated method/function.

Reply:
and what about recursive functions?